.drone.star

```diff
@@ -363,6 +363,12 @@ pipelineVolumeGo = \
         "temp": {},
     }
 
+ocisStorageVolume = \
```

Comment:
just to maintain consistency with above volumes. and including `step` helps to determine where to use this volume
```suggestion
stepVolumeOcisStorage = \
```